packages/fuels-test-helpers/src/service.rs

```diff
@@ -102,10 +103,14 @@ impl FuelService {
                 max_queries_resolver_recursive_depth: 1,
                 max_queries_directives: 10,
                 max_concurrent_queries: 1024,
+                required_fuel_block_height_tolerance: 10,
```

Comment:
Do we support historical execution when running the fuel core as a lib?

Reply:
https://github.com/FuelLabs/fuels-rs/pull/1628/commits/3cc45855c4dd38743053fd7e6a9b2c7b67e3a46e